src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -15,7 +15,8 @@ Since 7.17.0 Eclipse Che supports three types of storage with different capabili
 
 .Persistent storage
 
-This kind of storage gives you an ability to store your changes directly in the mounted Persistent Volume. In this case your changes will be totally safe and Kubernetes infrastructure takes care of it (storage backend in particular), but the price for that is slow I/O, especially with a huge amount of small files. For example, some NodeJS projects may have many dependencies and node_modules is filled with thousands of small files.
+This kind of storage gives you an ability to store your changes directly in the mounted Persistent Volume. In this case your changes will be totally safe and Kubernetes infrastructure takes care of it (storage backend in particular), but the price for that is slow I/O, especially with a huge amount of small files. For example, some NodeJS projects may have many dependencies and node_modules is filled with thousands of small files.  
+Speed of I/O can be various in different environment configuration, and depends on Storage Classes configured in your environment.More details can be reached here: link:https://kubernetes.io/docs/concepts/storage/storage-classes/[Storage Classes]
```

Comment:
I would remove `.More details can be reached here: link:https://kubernetes.io/docs/concepts/storage/storage-classes/[Storage Classes]` and add the link to the Storage Classes in the previous sentence.  Also maybe worth outlining it as a NOTE:

NOTE: Speed of I/O varies depending on the link:https://kubernetes.io/docs/concepts/storage/storage-classes/[Storage Classes] configured in the environment

Reply:
done